rotkehlchen/data/importer.py

```diff
@@ -96,7 +96,11 @@ def __init__(self, db: DBHandler) -> None:
         self.db = db
         self.db_ledger = DBLedgerActions(self.db, self.db.msg_aggregator)
 
-    def _consume_cointracking_entry(self, csv_row: Dict[str, Any]) -> None:
+    def _consume_cointracking_entry(
+        self,
+        csv_row: Dict[str, Any],
+        extra_parameters: Dict[str, Any],
```

Comment:
Okay so both here and in all other import functions, why add an `extra_parameters` dict that's making things more confusing and is not typed by mypy?

You added it to **all** import functions. Just add another argument called `timestamp_format`